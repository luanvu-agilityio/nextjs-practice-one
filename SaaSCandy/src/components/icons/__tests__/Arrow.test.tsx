import { render } from '@testing-library/react';
import ArrowIcon from '../Arrow';

describe('AddressIcon', () => {
  it('matches snapshot', () => {
    const { container } = render(<ArrowIcon />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
