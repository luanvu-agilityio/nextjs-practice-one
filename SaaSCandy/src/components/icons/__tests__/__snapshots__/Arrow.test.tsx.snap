// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`AddressIcon matches snapshot 1`] = `
<svg
  fill="none"
  height="20"
  viewBox="0 0 28 20"
  width="28"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M0.5 19C6.4482 18.8959 19.902 15.1572 26.1314 1.03594M17.9022 3.56138L26.3255 1L27.5 8.73063"
    stroke="black"
  />
</svg>
`;
